TEAM.md

```diff
@@ -6,8 +6,8 @@ Role | Members
 ---|---
 PM | @dmarkov
 CR | @darkled, @caarlos0, @carlosmiranda, @dmzaytsev, @pinaf, @ggajos, @hdouss, @krzyk, @mkordas, @longtimeago
-DEV | @gmandnepr, @piddubnyi, @jrdalpra, @amihaiemil, @nikoladespotoski, @prondzyn, @shelan, @yamstranger, @bertram25
-IMP | @gmandnepr, @piddubnyi, @jrdalpra, @amihaiemil, @nikoladespotoski, @prondzyn, @shelan, @yamstranger, @bertram25
+DEV | @gmandnepr, @piddubnyi, @jrdalpra, @amihaiemil, @nikoladespotoski, @nesterov-n, @prondzyn, @shelan, @yamstranger, @bertram25
```

Comment:
@mkordas are you sure? I simply merged with master before comitting, so this branch will have everything that master has.